meta-tx/test/MetaTransaction.test.js

```diff
@@ -78,13 +80,37 @@ const getTransactionData = async (nonce, abi, params) => {
   return {r, s, v, functionSignature};
 }
 
+async function areMetaTxSupported(addressContract) {
+  let nonce;
+	try {
+	  nonce = await addressContract.getNonce.call(publicKey);
+	} catch {
+     return(false);
+  }
+
+	let {
+    r,
+    s,
+    v,
+    functionSignature
+  } = await getTransactionData(nonce, getNonceAbi, [ZERO_ADDRESS]);
+
+  try {
+    await addressContract.executeMetaTransaction.call(publicKey, functionSignature, r, s, v);
+  } catch {
+    return(false);
+  }
+  return(true);
+}
+
 contract("ERC721MetaTxTokenTestAllien", accounts => {
   let erc721NoMetaTx;
   let metaTxTest;
   let owner = accounts[0];
 
   beforeEach(async () => {
     metaTxTest = await deployProxy(MetaTxTest, ["MetaTxTest", "1"], { initializer: '__MetaTxTest_init' });
+    noMetaTxTest = await NoMetaTxTest.new();
```

Comment:
this contract needed in 1 test, so it's better to deploy it there only

Reply:
ok, done